purview/api/v1.py

```diff
@@ -17,3 +21,20 @@ def about():
     :return: A JSON object with version info
     """
     return jsonify({'version': version})
+
+
+@api_v1.route('/<resource>/<uid>')
+def get_resource(resource, uid):
+    model_names = [model.__name__.lower() for model in all_models]
```

Comment:
This can actually be defined just before the error message. It saves the route from having to cycle through the list unnecessarily.